src/testrender/testrender.cpp

```diff
@@ -221,62 +225,69 @@ main (int argc, const char *argv[])
+        double setuptime = timer.lap();
 
         if (warmup)
             rend->warmup();
-        double warmuptime = timer.lap ();
+        double warmuptime = timer.lap();
 
         // Launch the kernel to render the scene
         for (int i = 0; i < iters; ++i)
-            rend->render (xres, yres);
-        double runtime = timer.lap ();
+            rend->render(xres, yres);
+        double runtime = timer.lap();
 
-        rend->finalize_pixel_buffer ();
+        rend->finalize_pixel_buffer();
 
         // Write image to disk
-        if (Strutil::iends_with (imagefile, ".jpg") ||
-            Strutil::iends_with (imagefile, ".jpeg") ||
-            Strutil::iends_with (imagefile, ".gif") ||
-            Strutil::iends_with (imagefile, ".png")) {
+        if (Strutil::iends_with(imagefile, ".jpg")
+            || Strutil::iends_with(imagefile, ".jpeg")
+            || Strutil::iends_with(imagefile, ".gif")
+            || Strutil::iends_with(imagefile, ".png")) {
             // JPEG, GIF, and PNG images should be automatically saved as sRGB
             // because they are almost certainly supposed to be displayed on web
             // pages.
-            ImageBufAlgo::colorconvert (rend->pixelbuf, rend->pixelbuf,
-                                        "linear", "sRGB", false, "", "");
+            ImageBufAlgo::colorconvert(rend->pixelbuf, rend->pixelbuf, "linear",
+                                       "sRGB", false, "", "");
         }
-        rend->pixelbuf.set_write_format (TypeDesc::HALF);
-        if (! rend->pixelbuf.write (imagefile))
+        rend->pixelbuf.set_write_format(TypeDesc::HALF);
+        if (!rend->pixelbuf.write(imagefile))
             rend->errhandler().errorfmt("Unable to write output image: {}",
                                         rend->pixelbuf.geterror());
         double writetime = timer.lap();
 
         // Print some debugging info
         if (debug1 || runstats || profile) {
             std::cout << "\n";
-            std::cout << "Setup : " << OIIO::Strutil::timeintervalformat (setuptime,4) << "\n";
-            std::cout << "Warmup: " << OIIO::Strutil::timeintervalformat (warmuptime,4) << "\n";
-            std::cout << "Run   : " << OIIO::Strutil::timeintervalformat (runtime,4) << "\n";
-            std::cout << "Write : " << OIIO::Strutil::timeintervalformat (writetime,4) << "\n";
+            std::cout << "Setup : "
+                      << OIIO::Strutil::timeintervalformat(setuptime, 4)
+                      << "\n";
+            std::cout << "Warmup: "
+                      << OIIO::Strutil::timeintervalformat(warmuptime, 4)
+                      << "\n";
+            std::cout << "Run   : "
+                      << OIIO::Strutil::timeintervalformat(runtime, 4) << "\n";
+            std::cout << "Write : "
+                      << OIIO::Strutil::timeintervalformat(writetime, 4)
+                      << "\n";
             std::cout << "\n";
```

Comment:
This was more readable prior to clang-format.

Reply:
This will be rewritten anyway as I switch all this stream output to the new format style, so will leave for now.